server/views/importfromgithub.py

```diff
@@ -18,8 +18,12 @@ def import_from_github(request):
 
     try:
         clientside_module = import_module_from_url(request.data["url"])
```

Comment:
Please make `import_module_from_url()` return a `ModuleZipfile`. Then, instead of hitting the module registry, pass `{module_zipfile.module_id: module_zipfile}`